scripts/build.py

```diff
@@ -713,6 +728,9 @@ def generate_copy_tasks_to_copy_from_source_to_target(
             if not any(
                     source_path.endswith(p) for p in FILE_EXTENSIONS_TO_IGNORE):
                 target_path = source_path
+                # Path in hashes is in posix style, if we are on Windows,
```

Comment:
Path in hashes --> The path in the hashes.json file

In general, for the explanation, explain why the constraints in this situation require the path to be in posix style, rather than why we need to convert it on Windows. (The second part is self-explanatory once you explain the first part.) See the example above I gave for the Java case -- it doesn't mention Windows at all.

Reply:
Updated